modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/ZonePartitionReplicaListener.java

```diff
@@ -273,6 +273,15 @@ public void addTableReplicaListener(TablePartitionId partitionId, Function<RaftC
         replicas.put(partitionId.tableId(), replicaListener.apply(raftClient));
     }
 
+    /**
+     * Remove table partition listener by table replication identifier from the current zone replica listener.
+     *
+     * @param partitionId Table partition id.
+     */
+    public void removeTableReplicaListener(TablePartitionId partitionId) {
```

Comment:
Shall we pass `int tableId` here instead?

Reply:
Done